src/desktop/apps/consign/components/create_account/__tests__/CreateAccount.jest.tsx

```diff
@@ -92,51 +79,118 @@ describe("React components", () => {
     expect(props.updateAuthFormStateAndClearErrorAction).toBeCalledWith("login")
   })
 
-  it("#handleSubmit calls props.handleSubmit with expected args", () => {
-    const wrapper = mount(
-      <UnconnectedCreateAccount {...props} />
-    ).instance() as UnconnectedCreateAccount
-    wrapper.handleSubmit(
-      {
-        email: "user@email.com",
-        password: "mypassword",
-      },
-      {}
-    )
-    expect(handleSubmitMock).toBeCalledWith(
-      "login",
-      {
-        contextModule: "consignSubmissionFlow",
-        copy: "Log In",
-        intent: "consign",
-        redirectTo: "/consign/submission",
-      },
-      { email: "user@email.com", password: "mypassword" },
-      {}
-    )
-
-    const wrapperWithTrackingParams = mount(
-      <UnconnectedCreateAccount {...props} contextPath="foo" subject="bar" />
-    ).instance() as UnconnectedCreateAccount
+  describe("with query params", () => {
```

Comment:
**thought [non-blocking]:** indirect testing?

I could be mistaken, but I think that the following tests are meant to cover how we compute the `redirectUrl`. They are integration tests because they use `mount` and trigger events.

I think this is probably overkill when what we really want is something more direct:

```ts
const component = new CreateAccount(props)
expect(component.redirectUrl).toEqual("/consign/submission?blah&blah&blah")
```

I marked this as non-blocking because I don't feel sure about the life expectancy of this code, so maybe this feedback doesn't matter. 😝 

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
This mostly follows existing conventions in the test, except I moved things into a describe block to better group (since a few more query param tests were needed). 

When the new designs for this page arrive and we move everything into relay lets be sure to think along these suggested lines tho 👍